src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -15,7 +15,7 @@
  */
 public class HelpWindow extends UiPart<Stage> {
 
-    public static final String USERGUIDE_URL = "https://se-education.org/addressbook-level3/UserGuide.html";
+    public static final String USERGUIDE_URL = "https://nus-cs2103-ay2223s1.github.io/tp/UserGuide.html";
```

Comment:
```suggestion
    public static final String USERGUIDE_URL = "https://ay2223s1-cs2103t-w11-3.github.io/tp/UserGuide.html";
```

Should this be our repo's user guide instead? :P